frontend/src/components/AppHeader.vue

```diff
@@ -1,21 +1,53 @@
-<script setup lang="ts"></script>
+<script setup lang="ts">
+import AppButton from "@/components/AppButton.vue";
+
+function addSapphireToMetaMask() {
+  if (!window.ethereum?.request) {
+    return alert(
+      "Have you installed MetaMask yet? If not, please do so.\n\nComputer: Once it is installed, you will be able to add the Sapphire ParaTime to your MetaMask.\n\nPhone: Open the website through your MetaMask Browser to add the Sapphire ParaTime."
```

Comment:
Let's try to avoid the "ParaTime" term where possible. Also, the "network" term is found in the MetaMask UIs.

```suggestion
      "Have you installed MetaMask yet? If not, please do so.\n\nComputer: Once it is installed, you will be able to add the Sapphire network to your MetaMask.\n\nPhone: Open the website through your MetaMask Browser to add the Sapphire network."
```